Lib/idlelib/configdialog.py

```diff
@@ -1844,6 +1845,9 @@ def create_page_general(self):
                 frame_context: Frame
                     context_title: Label
                     (*)context_int: Entry - context_lines
+                frame_cursor_blink_default: Frame
```

Comment:
1. Remove 'default' from comment lines and all code lines, including pack and tracer lines.
The line_number stuff is not an example to follow (I will change it also in a separate PR).

2. Move comment lines to frame_window, as this applies to Shell, Editor, and Output windows.
I want to try putting this just after win_size.

3. Move code similarly.